doc/developer/frr-release-procedure.rst

```diff
@@ -80,104 +82,148 @@ FRR Release Procedure
+#. Coordinate with the maintainer of FRR Debian package to publish the Debian
+   packages on that repository. Update the repository webpage. Verify that the
+   instructions on the webpage work and that FRR is installable from the
+   repository on a Debian system.
 
-15. Contact the current Debian maintainer for FRR to get new Debian packages
-    built and published on our APT repository at https://deb.frrouting.net/.
-    Ensure the webpage text is updated. Verify that new packages install
-    successfully on a vanilla Debian installation using the instructions on the
-    webpage.
+   Current maintainer: *Jafar Al-Gharaibeh*
 
-16. Deploy Snapcraft release (after CI system finishes the tests for snapcraft
-    testplan).
+#. Log in to the Read The Docs instance. in the "FRRouting" project, navigate
+   to the "Overview" tab. Ensure there is a ``stable-<version>`` version listed
+   and that it is enabled. Go to "Admin" and then "Advanced Settings". Change
+   "Default version" to the new version. This ensures that the documentation
+   shown to visitors is that of the latest release by default.
 
-17. Update the Read The Docs instance to being publishing documentation built
-    off the ``stable/<version>`` branch. Contact Quentin Young for this step.
+   This step must be performed by someone with administrative access to the
+   Read the Docs instance.
 
-18. Publish the GitHub release.
+#. On GitHub, go to the <https://github.com/FRRouting/frr/releases>_ and click
+   "Draft a new release". Write a release announcement. The release
+   announcement should follow the template in
+   ``release-announcement-template.md``, located next to this document. Check
+   for spelling errors, and optionally (but preferably) have other maintainers
+   proofread the announcement text.
 
-19. Clone the ``frr-www`` repository:
+   Do not attach any packages or source tarballs to the GitHub release.
 
-    .. code-block:: console
+   Publish the release once it is reviewed.
 
-       git clone https://github.com/FRRouting/frr-www.git
+#. Deploy Snapcraft release. Remember that this will automatically upgrade Snap
+   users.
 
-20. Add a new release announcement, using a previous announcement as template:
+   Current maintainer: *Martin Winter*
 
-    .. code-block:: console
+#. Build and publish the Docker containers.
 
-       cp <old-version>-launch.md <version>-launch.md
+   Current maintainer: *Quentin Young*
 
-    Paste the GitHub release announcement text into this document, and **remove
-    line breaks**. In other words, this::
+#. Clone the ``frr-www`` repository:
+
+   .. code-block:: console
+
+      git clone https://github.com/FRRouting/frr-www.git
+
+#. Add a new release announcement, using a previous announcement as template:
+
+   .. code-block:: console
 
-       This is one continuous
-       sentence that should be
-       rendered on one line
+      cp <old-version>-launch.md <version>-launch.md
```

Comment:
while doing this step I noticed that names were <version>.md,   no "launch" in the  name